### PR TITLE
Shopping Disaster

### PR DESCRIPTION
**Pull request by winniepee**
Map for CTF

### CODE REVIEW

**Comment by AnyBlue** [comment]
Where is map.mts and screenshot?

**Comment by winniepee** [comment]
[shopping_disaster.zip](https://github.com/MT-CTF/maps/files/5340027/shopping_disaster.zip)

**Comment by AnyBlue** [comment]
Just add to the Pull Request map.mts and screenshot for the map

File: shopping_disaster/map.conf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+chests.1.to = (-631,19,418)
+chests.2.n = 12
+chests.2.from = (-638,19,435)
+chests.2.to = (-646,19,417)
+chests.3.n = 18
+chests.3.from = (-582,19,435)
+chests.3.to = (-597,19,418)
+chests.4.n = 12
+chests.4.from = (-567,19,435)
+chests.4.to = (-575,19,418)
```
Comment (JostP) [inline shopping_disaster/map.conf:26]: I am sure this won't work

**Comment by JostP** [comment]
You will have to create new PR for this map. Make sure you create new branch from master before uploading files.

Zaslonska slika 2020-12-05 17-28-13

**Comment by Panquesito7** [comment]
_Offtopic_: If you want your forks to be updated with the main repository, the [Pull](https://github.com/apps/pull) app may be useful. 🙂

**Comment by Avyukt1** [comment]
_Offtopic/Tip_: Never use the base branch of the repository for Pull Requests. don't use the `master` branch as its the base branch for [Maps](https://github.com/MT-CTF/maps). create a new branch or that will cause lots of errors That are in your other PRs.

**Review by Panquesito7 — COMMENTED** [review]
The target repository for this PR has been deleted.

**Comment by Avyukt1** [comment]
Can't test the map.. There are many bugs in the `map.conf`

**Comment by Avyukt1** [comment]
Adopted the changes.
Use [The New Pull Request](https://github.com/MT-CTF/maps/pull/182) instead of this.

**Pull request closed without merging**